.travis.yml

```diff
@@ -12,8 +12,8 @@ branches:
 jobs:
   include:
     - stage: Lint & Test
+    - script: ./hack/tf-lint.sh
```

Comment:
The first entry in after `- stage` doesn't need a hyphen (so `- script` -> `script` here).  The hyphen you're adding is causing Travis to run [this dummy job][1].

[1]: https://travis-ci.org/openshift/installer/jobs/411958060#L380